clustered/client/src/test/java/org/ehcache/clustered/client/replication/ActivePassiveClientIdTest.java

```diff
@@ -136,34 +138,28 @@ public void tearDown() throws Exception {
 
   @Test
   public void messageTrackedAndRemovedWhenClientLeaves() throws Exception {
-    Set<ClientSourceId> clientSourceIds = activeMessageHandler.getTrackedClients();
-    assertThat(clientSourceIds).hasSize(1); // one client tracked
-
-    Map<Long, EhcacheEntityResponse> responses = activeMessageHandler.getTrackedResponses(clientSourceIds.iterator().next());
-    assertThat(responses).hasSize(0); // no message tracked right now
+    Assert.assertThat(activeMessageHandler.getTrackedClients().count(), is(0L)); // no client tracked
 
     storeProxy.getAndAppend(42L, createPayload(42L));
 
+    Map<Long, EhcacheEntityResponse> responses = activeMessageHandler.getTrackedResponsesForSegment(11, activeMessageHandler.getTrackedClients().findFirst().get());
```

Comment:
11 is where the message magically ends up? That's forever stable? Can you do a constant for it? To make it more obvious what it is by giving it a name.

Reply:
Yeah, did not come up with a way of computing it easily